src/scenes/DeliverablesRouter/PageMessage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
Thoughts on a more descriptive filename? RejectedDeliverableMessage?

Reply:
I do agree that `PageMessage` feels a bit too generic and in my mind could be confused to be a more general purpose component. `RejectedDeliverableMessage` seems reasonable, I'll rename it to that then merge the PR.